src/ak_schema/schema/ak_specimens.yaml

```diff
@@ -19,23 +19,53 @@ classes:
     slots:
       - specimen
 
+  SpecimenProcessing:
+    is_a: PlannedProcess
+    class_uri: OBI:0000094 # material processing
+    slots:
+      - specimen
+
   Assay:
-    is_a: NamedThing
+    is_a: PlannedProcess
     class_uri: OBI:0000070 # assay
     slots:
       - specimen
-      - type
+      - specimen_processing
       - assay_type
```

Comment:
I was using `assay_type` for the OBI ontology ID of the assay, which could be more specific than the `class_uri: OBI:0000070` for the `Assay` class. Note that ontology class of the assay is about the meaning, while the LinkML class is (more) about the syntactic structure (e.g. which slots it has).

We could get rid of this if we're happy with one LinkML class per ontology class, but that's not how I had been thinking about it.

Reply:
Gotcha, added this to the comments for clarity.